interp/notation_ops.ml

```diff
@@ -1570,10 +1587,10 @@ let rec match_cases_pattern metas (terms,termlists,(),() as sigma) a1 a2 =
  match DAst.get a1, a2 with
   | r1, NVar id2 when Id.List.mem_assoc id2 metas -> (bind_env_cases_pattern sigma id2 a1),(false,0,[])
   | PatVar Anonymous, NHole _ -> sigma,(false,0,[])
-  | PatCstr ((ind,_ as r1),largs,Anonymous), NRef (GlobRef.ConstructRef r2) when Construct.CanOrd.equal r1 r2 ->
+  | PatCstr ((ind,_ as r1),largs,Anonymous), NRef (GlobRef.ConstructRef r2,_) when Construct.CanOrd.equal r1 r2 ->
```

Comment:
I did not explore all situations but I would feel more comfortable with `None` (or `None` or `Some (UAnonymous _)`). Conversely, is there a situation where we would like an explicit universe?

Same remark below in the three other situations.

Reply:
Done.